api/mock/facade.go

```diff
@@ -26,7 +26,7 @@ type Facade struct {
 	GenerateTransactionHandler func(sender string, receiver string, value *big.Int, code string) (*transaction.Transaction, error)
 	GetTransactionHandler      func(hash string) (*transaction.ApiTransactionResult, error)
 	CreateTransactionHandler   func(nonce uint64, value string, receiverHex string, senderHex string, gasPrice uint64,
-		gasLimit uint64, data string, signatureHex string, chainID string) (*transaction.Transaction, []byte, error)
+		gasLimit uint64, data string, signatureHex string, chainID string, version uint32) (*transaction.Transaction, []byte, error)
```

Comment:
do not forget to expose the minimum transaction version on the network config route (in order to be used by the frontend). Also, create PR to update the proxy/txgen repos. Frontend should be upgraded too.

Reply:
Add unit tests all over the PR where the new data field are verified.

Reply:
Already exposed minimum transaction version in api route `/network/config`. And  PR for proxy and tx-gen will be open soon (work in progress).

Reply:
more unit tests was added